samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,377 @@
+
+Presentation delay
+    The receiver(s) of isochronous data may receive their data at different points in time, but need to render/present it synchronously.
+    Presentation delay represents the amount of time that is added to the received timestamp to obtain the presentation time of the data.
+    In this sample the presentation delay is configured with the Kconfig option :kconfig:option:`CONFIG_TIMED_LED_PRESENTATION_DELAY_US`.
+
+For more details about this feature, see `Bluetooth Core Specification Version 5.2 Feature Overview <https://www.bluetooth.com/wp-content/uploads/2020/01/Bluetooth_5.2_Feature_Overview.pdf>`_.
+
+This sample demonstrates the following:
+
+* Any of the possible isochronous topologies with unidirectional data transfer.
+  The topology is selected when the application starts.
+* The transmitter of the data sends it over all configured and connected streams.
+  The number of streams is configured with the :kconfig:option:`CONFIG_BT_ISO_MAX_CHAN` Kconfig option.
+* The receiver of the isochronous data receives data from one transmitter only.
+* The maximum transport latency and retransmission count is configured when the application starts.
+
+
+Sample structure
+================
+
+The sample code is divided into multiple source files which makes it easier to compile in or out certain functionality.
+
+``main.c``
+  The main entry point of the sample allows the user to selects the role and parameters to be used.
+
+``bis_transmitter.c``
+  This file implements a device that sets up a BIS transmitter with a configured number of streams.
+  That is:
+
+  1. Configure and start an extended advertiser.
+     The extended advertiser advertiser data contains the device name.
+  2. Configure and start a periodic advertiser.
+  3. Configures and starts a broadcast isochronous group (BIG).
+
+``bis_receiver.c``
+  This file implements a device that syncs to a BIS broadcaster with a given BIS index.
+
+  1. Scan for extended advertisers containing the device name.
+  2. Synchronize to the corresponding periodic advertiser.
+  3. Synchronize to the selected BIS.
+
+  When the ISO channel is disconnected, it tries to sync to it again.
+
+``cis_central.c``
+  This file implements a device that acts as a CIS central.
+  The central can either be configured as a transmitter or a receiver.
+
+  1. Configures a connected isochronous group (CIG).
+  2. Scans and connects to a device that contain the device name.
+  3. Sets up an CIS to the connected device.
+  4. Continues to scanning for more peripherals if there are more available isochronous streams.
+
+``cis_peripheral.c``
+  This file implements a device that acts as a CIS peripheral.
+  The peripheral can either be configured as a transmitter or a receiver.
+
+  1. Sets up an isochronous server so that it can later on accept setup or a CIS.
+  2. Starts advertising its device name.
+  3. Once a device has connected, accept the incoming CIS request.
```

Comment:
```suggestion
  * Set up an isochronous server so that it can later on accept the setup or an CIS.
  * Start advertising its device name.
  * Once a device has connected, accept the incoming CIS request.
```